requirements/ingest-s3.txt

```diff
@@ -12,16 +22,16 @@ argilla==1.3.1
     # via
     #   -r requirements/base.txt
     #   unstructured (setup.py)
+async-timeout==4.0.2
+    # via aiohttp
+attrs==22.2.0
+    # via aiohttp
 backoff==2.2.1
     # via
     #   -r requirements/base.txt
     #   argilla
-boto3==1.26.80
-    # via unstructured (setup.py)
-botocore==1.29.80
-    # via
-    #   boto3
-    #   s3transfer
+botocore==1.27.59
```

Comment:
interesting that s3fs wants a significantly older version of botocore, published Aug 24, 2022 🤔

Reply:
So AFAIU `s3fs` is actively maintained (see https://github.com/fsspec/s3fs) and widely use by some popular libraries such as `transformers` or `datasets`, since it's easy to plug in new remote data sources based on `fsspec`.